start

```diff
@@ -426,6 +434,12 @@ function init_soroban_rpc() {
   perl -pi -e "s/__NETWORK__/$NETWORK_PASSPHRASE/g" $SOROBAN_RPC_CAPTIVE_CORE_CFG
   run_silent "finalize-core-config-enable-soroban-diagnostic-events" perl -pi -e "s/__ENABLE_SOROBAN_DIAGNOSTIC_EVENTS__/$ENABLE_SOROBAN_DIAGNOSTIC_EVENTS/g" $SOROBAN_RPC_CAPTIVE_CORE_CFG
   
+  ADMIN_ENDPOINT=0.0.0.0:8004
```

Comment:
`ADMIN_ENDPOINT` should be a local variable:
```suggestion
  local ADMIN_ENDPOINT=0.0.0.0:8004
```

Reply:
ah, haven't used that in shell before, thanks.